content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md

```diff
@@ -49,3 +49,81 @@ table th:nth-of-type(2) {
 ## Known Limitations and Issues
 
 There is no support for upgrades from the early access beta versions.
+
+## Test: Table in HTML Format
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:"Poppins", -apple-system, blinkmacsystemfont;font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:"Poppins", -apple-system, blinkmacsystemfont;font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-g9j2{background-color:##efefef;border-color:#c0c0c0;font-weight:bold;text-align:left;vertical-align:top}
+.tg .tg-f58o{background-color:##efefef;border-color:#c0c0c0;font-weight:bold;text-align:left;vertical-align:bottom}
+.tg .tg-wo29{border-color:#c0c0c0;text-align:left;vertical-align:top}
+</style>
+<table class="tg" style="undefined;table-layout: fixed; width: 920px">
```

Comment:
`<table class="tg" style="undefined;table-layout: fixed; width: 920px">`
to 
`<table>`